run.py

```diff
@@ -295,12 +297,18 @@ def _validate_profile_options(profile_options: str):
         exit(-1)
 
     found = False
-    Model = load_model_by_name(args.model)
-    if not Model:
-        # try load model from canary
-        Model = load_canary_model_by_name(args.model)
-        if not Model:
-            print(f"Unable to find model matching {args.model}.")
+
+    try:
+        Model = load_model_by_name(args.model)
+    except ModuleNotFoundError:
+        traceback.print_exc()
+        exit(-1)
+    except:
+        traceback.print_exc()
```

Comment:
No need to print stack backtrace if it is model not found.

Reply:
cc @JansonJohnPZ Can you remove the stacktrace print when we met "ModelNotFoundError"?